citesphere/src/main/webapp/WEB-INF/views/auth/authorities/create.html

```diff
@@ -329,8 +328,11 @@ <h1>Create Authority</h1>
         <td><input id="uri" th:field="*{uri}" class="form-control" readonly/></td>
     </tr>
     
-    <input id="importerId" th:field="*{importerId}" class="form-control" />
-    
+    <tr id="tr-importerId"> 
+        <td width="20%"><label for="importerId">Importer Id</label></td>
+        <td><input id="importerId" th:field="*{importerId}" class="form-control" readonly/></td>
```

Comment:
I think this will lead to the wrong importer id, doesn't it? It says "Conceptpower" and this will be set as importer id in the controller.